Goal: Correct the gwc.1 man page for a syntax error

Status wrt upstream: Patch to propose

Index: geneweb-4.10/man/gwc.1
===================================================================
--- geneweb-4.10.orig/man/gwc.1	2002-11-01 11:35:53.000000000 +0100
+++ geneweb-4.10/man/gwc.1	2005-09-01 18:35:29.000000000 +0200
@@ -48,7 +48,7 @@
 .BI \-sep
 Separate all persons in next file
 .TP
-.BI \-sh  \f\<int\>\f
+.B \-sh <int>
 Shift all persons numbers in next files
 .TP
 .BI \-ds 
